applications/ContactStructuralMechanicsApplication/custom_utilities/derivatives_utilities.h

```diff
@@ -533,7 +531,7 @@ class DerivativesUtilities
         const DecompositionType& DecompGeom,
         const PointType& LocalPointDecomp,
         const PointType& LocalPointParent,
-        const bool ConsiderNormalVariation = false
+        const NormalDerivativesComputation ConsiderNormalVariation = NODERIVATIVESCOMPUTATION
```

Comment:
NO_DERIVATIVES_COMPUTATION would be better

Reply:
Ok